### PR TITLE
chore(ci/logging): merge with stdlib

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
Also default the runner to a medium instance rather than small.

## Reviewer Checklist
- [ ] Title is accurate.
- [ ] Description motivates each change.
- [ ] No unnecessary changes were introduced in this PR.
- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Release note has been added for fixes and features, or else `changelog/no-changelog` label added.
- [ ] All relevant GitHub issues are correctly linked.
- [ ] Backports are identified and tagged with Mergifyio.

### CODE REVIEW

**Comment by Kyle-Verhoog** [comment]
hmm this slowed down the run-time substantially, all the way to 15mins now.

File: .circleci/config.yml [hunk 1]
```diff
@@ -579,17 +579,11 @@ jobs:
...
           pattern: 'ddtracerun'
 
-  test_logging:
-    <<: *contrib_job_small
-    steps:
-      - run_test:
-          pattern: 'test_logging'
-
   stdlib:
     <<: *contrib_job_small
```
Comment (brettlangdon) [inline .circleci/config.yml:583]: parallelism 2 ?

File: .circleci/config.yml [hunk 2]
```diff
@@ -578,17 +578,11 @@ jobs:
...
 
-  test_logging:
-    <<: *contrib_job_small
-    steps:
-      - run_test:
-          pattern: 'test_logging'
-
   stdlib:
-    <<: *contrib_job_small
+    <<: *contrib_job
```
Comment (brettlangdon) [inline .circleci/config.yml:582]: I mean, you went from 4 nodes for 4 suites to 1 node for 4 suites to 4 nodes for 5 suites ?

Comment (Kyle-Verhoog) [reply .circleci/config.yml:582]: lol yes, idk why the logging tests take so long, if it weren't for them then it'd be fine 🤔

**Comment by Kyle-Verhoog** [comment]
gonna defer this for now, the logging tests are taking too long (seems to just be due to subprocess test cases... idk why those take ~1s each).

**Pull request closed without merging**